### DISCUSSION TITLE
Thinking Cap #6

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#32] [Assignments]
### Reminder: Responses submitted after Sunday 2pm will not be eligible for awards.

> ## How does data justice and design justice relate? How have you considered both in relation to your group projects?

The reading on [Data Justice](https://albertkun.github.io/22S-ASIAAM-191A/materials/readings/DataJustice.pdf) will be important to completing this thinking cap.

**Comment by NishuKrish** [comment]
Data justice and design justice both relate in the sense that seek to empower communities with the tools of technologies, and to mitigate the harmful impacts these technologies can have. They try and center the community over the actual product or tool itself. It is important for any project to keep the community in mind when designing the site (with regards to making it simple and accessible) and also to make sure that any data that is being used in handled in a way that protects the person's privacy and cannot be used to harm them. 
With regards to our project, we have considered the ideas of design/data justice and intend to use it in order to make our site more accessible for the community we are focusing on, Asian American students at UCLA. and their academic experiences  More importantly, we want to use these principles to make it so that the experiences of Asian American students are the focus of the site, rather than the project itself. With regards to design justice these are some of the main points I have thought about:
1. Layout of the site - The layout should not be confusing and littered with extraneous material. Our map will be at the forefront, and we will have an easily accessible button for someone to access our survey. 
2.  Ease of Survey- We intend to make our survey accessible by limiting the number of questions we have so that people do not become confused or fatigued. We also make the questions themselves easy to read/open ended. We tried to make them not too complex as then it becomes hard for someone to answer our question. 
With regards to Data Justice, we intend to:
1. Make sure not to collect any personal information beyond the scope of what is required, 
2.  Providing a data delete option for someone to contact us should they change their mind and not want to share their experiences
3. Clearly state at the beginning of our survey what we are going to do with the responses that are being collected.

**Reply by daikis12** [reply]
I agree that both design justice and date justice center on the people and community over the product or tool itself, which I believe is an essential concept to keep in mind for our project. In our project, we are told to listen to the people we are surveying and empower them by creating a platform where they can share their experiences and concerns. This is less about us finding solutions for them but more about giving people a place to express themselves, which I believe is very important.

**Reply by codingdianat** [reply]
Similar to your project, my project also aims to explain the experiences of UCLA students. I find it really impressive that you connect every aspect of your website to the community. I am especially impressed with your simple website layout, which will boost accessibility for users. After reflecting on my project as well as your discussion, I realized my survey questions are on the longer side and may cause feelings of fatigue that you mentioned.

**Comment by daikis12** [comment]
Data justice and design justice are related to each other as they both emphasize the importance of data and design in relation to social issues. In this week’s article, data is explained “as something that is situated and necessarily understood in relation to other social practices” (p. 873). Technology has become an essential part of everyday human lives, meaning that it has become an integral part of society. This makes it crucial for people to understand what and how data and design are represented. As some examples from last week and this week’s readings explain, some communities or groups of people (often marginalized) are not well represented in the data and design we see every day because of unequal power relations in our society. Both data justice and design justice attempt to tackle some of the most serious problems deeply rooted in our society by “pay[ing] particular attention to structural inequality, highlighting the unevenness of implications and experiences of data [and design] across different groups and communities in society” (p. 875). In relation to our group project, we follow some of the principles of both data justice and design justice by focusing on empowering the community that we’ve chosen. In order to do so, we build our page that is easy to access and is useful for effectively conveying our participants’ stories to other users, instead of focusing too much on aesthetics or professionalism.

**Reply by codingdianat** [reply]
I like that you specifically state that the goal of your webpage is to empower the community of your choice. I think having a clear goal of what community you want to help will definitely make the design process easier. I like that your focus is not on aesthetics but on user needs.

**Reply by kericsp** [reply]
I like that you emphasize how we are in an era in which digital and data technology has been integrated into the human experience, and how this dictates the importance of justice in both data and design. I'm excited to see how you incorporate these principles into your final project!

**Comment by codingdianat** [comment]
Although data and design are separate fields that seem drastically different from each other, they are both tools that play a crucial role in maintaining justice. Design justice and data justice both aim to shift the focus away from creating technological advancements to using these data and design tools to addressing the actual source of the problem. Instead of prioritizing technological advancement, we should be reflective of the “limitations of focussing on technology as the central component of contemporary injustices”. (P 877).  A key component of data and design justice is accepting that social issues need to be addressed on the human level, not through technology. As someone who aspires to work in tech, this important topic is not discussed enough. I personally believe that the tech field is currently prioritizing innovation over social issues.  In our project, we have considered design justice by prioritizing accessibility when designing our webpage. We acknowledge the different experiences that people may have on our website (due to color blindness etc) and design with them in mind. Our webpage will prioritize ADA users through our choice of colors and layouts. In addition, our website is meant to amplify the frustrations and emotions of those who visit our page. Therefore, we will put their voices as the forefront of our website. In order to practice data justice in our project, we will prioritize anonymity for our users. In addition, we will create a separate google form that will allow users to remove their entry if they choose to.

**Reply by shubhgupta2018** [reply]
Your discussion reminds me of [an article I recently read ](https://www.tandfonline.com/doi/pdf/10.1080/00472338085390041?casa_token=_9tFSlrIRc8AAAAA:l9sNakb9e7KdksCgwOQwlDRL_OkV6BFWGGOODf8sbOHEGDVVz8MrZ-gky-5Ew1VmYOOQkwWMjFXZ)related to the relationship between technology and fascism. Here's a specific quote from it:

_"The organization of the neo-fascist state is, in substantial part, the result of the insertion of part of the metropolitan state apparatuses - notably the military and police bureaucracies. The sustenance of these apparati by the advanced capitalist countries leads to their technological development and staffing far in advance of the rest of society. The "overdeveloped" features of the neo-fascist state is a prime example of the external alliance which nurtures it."_ (Petras 1980, p.120).

Your description of how we should decentralize prioritizing technological advancement reminds me of this quote because I think it's a really good reminder of how innovation for the sake of innovation is not intrinsically good. Instead, our technologies must be intentional in nature and execution, and it is our responsibility to critique the technologies we observe, use, and create.

**Reply by emmashanson** [reply]
Your explanation for the difference between design justice and data justice was very in depth and provided a comprehensive summary of the reading. As for some implementations into your group project, I found similarities with our group project about accessing for those who are color blind and thought that it was great. In addition to that, I also found the separate form to remove responses if wanted to be really smart and a great way to display data justice in an efficient manner, and further pushes ideas as to how to provide as accessible of a platform to all as possible.

**Comment by antonyblin** [comment]
I see data justice and design justice as being connected in that design justice is concerned with inequities in the presentation and accessibility of data, whereas data justice is concerned with inequities in and the exploitativeness of data collection, and neither do an equal system make. Innovation and advancement are crucial to the amelioration of society, but only when it is done equitably, which is why design and data justice exist. I have definitely considered how both of these will fit into my group’s project, but I’ve definitely struggled with the implementation of my thoughts; however, for basic accessibility purposes, we want to offer light and dark modes as well as the ability to translate website content into different languages. Additionally, we’ve put a lot of effort into trying to make sure that the UCLA community is being highlighted, rather than other institutions that are connected to the community.

**Reply by daikis12** [reply]
I agree with you that design justice and data justice are connected in a way that they both attempt to make data and design more accessible, building a more equitable society. As we read in the articles, marginalized communities are often excluded from the process of data collection and presentation, and since data and design are a ubiquitous and essential part of society, they should be accessible to anyone. I think it is a good idea to implement a translation feature on your website as it makes it more accessible to users.

**Reply by shubhgupta2018** [reply]
It was really great reading about how your group has implemented measures like light and dark modes and translation services to make the understanding of the data in and of itself more accessible. I had previously not really thought about that before, but I will definitely think more about website accessibility even more to ensure that the principles of design justice are upheld. Thank you for sharing your insights!

**Reply by emmashanson** [reply]
It was great reading your response since it provides some examples of your accessibility purposes of light versus dark mode, taking light sensitivity into account was not something that I had considered previously before but definitely made me reconsider otherwise into my group project. In addition, it was great seeing the distinction between the communities that are being highlighted versus the institutions that are responsible for the communities; it's a great way of showing data justice and further pushes the narrative.

**Reply by ntran27** [reply]
It's a seemingly minor detail but light and dark mode makes a huge difference. I was thinking through what other technical details would be implemented to make our website more accessible. One thing that comes to mind is how a screen reader might interpret our website design.

**Comment by shubhgupta2018** [comment]
I see the relationship between data justice and design justice to be related to epistemological analyses of knowledge derivation. Epistemology refers to the means by which we derive knowledge about the existence and state of being (otherwise known as ontology), and it primarily is used in a philosophical or higher education setting to reflect the need to carefully obtain knowledge in a means that maintains the integrity of lived experiences. In context of this, data justice and design justice are therefore intimately related while also occupying different spaces of epistemological analysis. Whereas data justice centers discussion on how information and data are collected, design justice emphasizes how to resist what Dr. Costanza-Chock refers to as the matrix of domination in the communication of said data to external users. In that sense, data justice and design justice function in a dialectic: to achieve the Black feminist notion of collective liberation, both must uphold and center community needs and narratives, and neither can be compromised if radical change is to be achieved.

To further illustrate the relationship between data justice and design justice, take predictive policing software like PredPol. Such technologies of surveillance have exacerbated violence against Black and Brown communities. Data justice would be concerned more with the data that predictive policing utilizes and try to answer questions related to the purpose and intention of collecting data like arrests per police beat. Data justice would also be concerned with what that data reflects in and of itself in relation to police brutality. On the other hand, design justice would focus more on how that data is utilized through software and thereby further weaponized against marginalized communities. Design justice would focus more on understanding how the code in programs like PredPol exacerbate harmful policy actions like increasing policing within Black and Brown neighborhoods. Moreover, design justice would inquire why police should be the users of this sort of data and critique the lack of accountability of softwares like PredPol.

For the UCLgAy project, we have implemented principles of data and design justice through multiple steps, including the following three actions:

1. Our project focuses on PrEP accessibility at UCLA, which is an extremely salient subject for queer people (particularly QPOC) since the 1980's AIDS epidemic.
2. Our group utilized and continues to utilizes the narrative of members of the UCLA undergraduate queer community (aka me) to ensure that this project actively centers community needs. 
3. By making our project descriptive rather than prescriptive, we ensure that community narratives are centered and that our positionality as critical cartographers does not compromise the collection or presentation of data.

**Reply by ananyakodali17** [reply]
Your example of PredPol was a really good way to explain the interconnection of data justice and design justice in a very (unfortunately) relevant context. I like how you emphasized the fact that your inherent biases will not be influencing the community narratives in your website, but rather that the project will be "descriptive rather than prescriptive." I think that is a great way to phrase the goal of our group projects!

**Comment by emmashanson** [comment]
During this week’s lecture, I had a key takeaway from the slides that said “If you want to design solutions, you have to be a facilitator first”. I think that is a great way to show the “justice” aspect of our project. Data justice is data captured by the community in order to reflect systemic harm, with the goal of trying to break out of that cycle. Design justice is very closely connected with it, bridging the current gaps that we see in modern technology all too often, with accessibility often being geared towards English speakers (as an example though there are a lot more that can be expanded on). For for my group project, I want to implement design justice by allowing the survey to be completed in Japanese as well to include everyone’s stories, and to have our markers and our visual data also be color-blind friendly/sensitive, and also overall have larger/bolded text depending on what is being covered so that it can be friendly to those with visual impairments.

**Reply by ananyakodali17** [reply]
I think that it is great that you have considered colorblindness in your website design, as well as language barriers! I agree with your takeaway from lecture as well, I think that it was a great point which summed up the essential topic well.

**Reply by spolcharoen** [reply]
I really liked how you referenced the lectures in your post! It is always interesting to see the connections between the lecture and the readings. Also, it is really cool that you are taking colorblindness into account for your website. Visual/color accessibility is very important!

**Comment by ananyakodali17** [comment]
Data justice and design justice are related in the sense of always considering the social issues and empowering the targeted communities in the process. Design justice focuses on ensuring that the presentation of data is equally accessible to everyone, factoring in things like lack of access to technology and language barriers, while data justice focuses on recognizing the inequalities in data collection and how that can be exploited. I think that both topics are equally important to include when designing websites and tools such as our group projects, as they are tools to ensure the proper community is empowered. In our project, we are committed to doing our best to use the principles of design and data justice in the following ways:
-Making our survey short with only relevant questions
-Making our survey completely anonymous and providing an option to delete responses
-Creating a non-cluttered website which clearly displays exactly what we want to be shown in a way that shows the data as received in our survey

**Reply by spolcharoen** [reply]
I agree that data justice and design justice are working towards a common goal in different ways. I also enjoyed reading how you are implementing these ideas into your project. I think factoring in UI/UX into the survey and website design is really important, and I like how you are making your survey anonymous!

**Reply by stefuntran** [reply]
I find a sense of commonality between our groups in that we need to make short and relevant questions to our users. I am curious to see if this is a result of us being in a traditional academic environment and using these formal forms compared to being out in these communities and knowing what to ask.

**Comment by bittertang** [comment]
Both data justice and design justice go back to the idea of centralizing power in the communities they’re intended to serve but in different ways. Data justice is about collecting information from the community and taking into consideration how the information will be used to uplift them, whereas design justice involves the experience these communities will have in using whatever application or project the data will be used for. In relation to social justice, these 2 concepts are about reshifting preexisting dominating narratives to focus on the voices of marginalized communities. In our group project, we aim to uphold data justice through the way we’re gathering data in our surveys. We framed and structured the questions in our survey in a way that focuses on listening to the community as opposed to trying to solve the problem of educational inequity. As for design justice, we aim to prioritize user experience by understanding how our users, which will be Asian American UCLA students, will be interacting with our site and thinking about how the layout should be visually presented.

**Reply by kericsp** [reply]
I like that you mentioned listening to the community and prioritizing the user experience rather than trying to create a prescriptive solution! Your response connects to this week's reading's emphasis on "data witnessing", which is ensuring that data is accurately representative of the community rather than from an outsider's perspective. Good job!

**Comment by spolcharoen** [comment]
Data justice and design justice are related in their overarching goals to empower communities and consider the repercussions and impact of tech. As outlined in the reading, both ideas turn to social justice and inequality issues, as well as usability frameworks highlighting user experience, to achieve this. Data justice tackles the back end, while design justice tackles the front. Having taken classes in programming as well as UI/UX, it is interesting to see the commonalities between the two when considering users and community impact. Data justice and design justice have similar goals, but also face similar struggles, as it is easy to get caught up in the creation of a product and forget about the functionality. 

For my group project, we have considered data justice and design justice in each step. When writing our memo, we really had to reflect on what community we wanted to empower, how we were going to do that, and also whether or not it was actually wanted/useful help. Furthermore, when we started creating our survey and survey diagram, we had to consider the user flow of the survey. We wanted to make the survey as accessible as possible and prevent survey fatigue. It has been a challenge to think about which questions are actually relevant/necessary to our goals. Also, adding ways to delete survey responses and give more choice back to the respondent was something we had to consider. Lastly, as we begin thinking of the wireframe and what our application will look like, we have to turn to UI/UX to make sure the right information is being displayed with respective importance, as well as how the interactions will work on our site.

**Reply by kris10fu** [reply]
I agree that data justice and design justice have similar struggles as they often relate to the pitfalls of how technology is seen and used in our modern society. Discussion of technology is centered around innovation rather than how it pertains to and harms social justice and equity. I like how you mentioned the challenges of creating and designing a survey as I never really considered issues relating to this process, particularly how to determine what questions are actually needed for your goal. I think being more aware of data justice and design justice when creating the website can bring up a lot of meaningful questions and ideas that can improve the site.

**Reply by ntran27** [reply]
Survey accessibility and fatigue are really good points. I think that sometimes the usability of our surveys can become an after thought because we might want to collect extensive information without considering the effort it would take on the part of the survey respondents.

**Comment by penelopelamb** [comment]
Both data justice and design justice prioritize the target community and its benefit over the product's profit or aesthetic appeal. In doing so, the goal of both data justice and design justice is to combat the potential for an unethical relationship between product and user. In our group project, we are constantly reminding ourselves to prioritize the target community. One way that we are implementing data justice and design justice into our project is through transparency with the community. The description of our survey includes an estimation for the duration of the survey, a description of how survey data will be used, and contact information for removal of data from our project. Moreover, we chose to minimize unnecessary survey questions, such as ones that might not apply to the entire target community, in order to avoid survey fatigue. Even though a side effect of doing so could result in a less complete profile for each respondent, we have discarded accessory questions that are not essential to the project.

**Reply by stefuntran** [reply]
I can relate to your point about reminding ourselves that the target community is the focus center and not anything else. I am still working on changing the way I think about these scenarios and have been getting better throughout the quarter. Also, taking out unnecessary survey questions is something that our group has been trying to do as well. This was done through either combining questions or questioning if it is necessary in general.

**Reply by willa-21** [reply]
Yes, survey fatigue is such a good point. Our team also struggled with narrowing down the questions we wanted to ask. In terms of data justice, it is helpful to think about how the questions might impact the community - in terms of how the data could be misconstrued or prevent anonymity.

**Comment by stefuntran** [comment]
Data justice and design justice intersect through the fact that they are both trying to improve minority and underrepresented communities. What they do with these important communities is the difference that separates them. Data justice is when people take information from these communities and actively listen to what they say, such as improved roads, healthcare, and education. Design justice encompasses making user designs that accommodate the various groups of people compared to those who make it.  For our group, design justice is considered heavily because our website wants to encourage individuals to report their story while not taking away from their emotions and perspective. If our platform design is not user friendly for them in regards to our objective, then that will deter them from wanting to do anything which is not the goal of our platform. Data justice is equally as important because our goal again is to retain the emotions that our user has. The survey questions or wording are critical to best report what the user wants to say and our group is working towards the goal!

**Reply by kris10fu** [reply]
I like how you distinctly compared design justice and data justice to one another, and I agree that design justice seems to talk more about user design and experience. I also think that it's important to make the websites user-friendly, both in terms of the actual design as well as the ways in which we are collecting their information. I like how you emphasized that you want to retain the emotions of the user as this is a central aspect of their personal story and experience. It's important not to unintentionally inflict our own emotions and opinions on the survey, as that can affect the data that is collected.

**Reply by kaittli** [reply]
Your point about how design can encourage more engagement in your website is very valid; there should definitely be more discussion on how some designs actually discourage people from interacting and sharing stories. Design that takes away or suppresses the emotional aspects of people stories is design that is not seeking to empower users. I agree that the framing of survey questions is also very important as if questions are worded in a biased manner, many answers that are received may not be accurate.

**Comment by kris10fu** [comment]
Data justice and design justice relate to each other by both centering around critiquing existing systems that uphold technological determinism, which mitigates the voices, stories, and participation of underserved and marginalized communities. Data justice may be seen as being more involved with the collection and utilization of data while design justice may be more concerned with the visualization and experience of data. Both seek to redefine the current relationship between society and technology, aiming to enhance goals related to social justice and equity. In the reading on Data Justice for this week, the authors emphasized how data needs to be understood in relation to other social practices, especially regarding how data analysis coincides with structural inequality. In last week’s reading on design justice, there was also a significant emphasis on how design cannot be experienced without the interconnecting issues of race, class, and gender. Thus, data justice and design justice inform and interact with each other as parallel movements focused on the complicated relationship between technology and social justice.

In terms of my group project, my group and I have considered data justice in terms of who we are collecting data from, and the form in which we are doing that collection. We aim to gather data directly from the local community members (i.e. the elderly residents of housing projects in Little Tokyo) which allow for data collection that reflects the actual experiences and needs of the people being affected. This ensures that our project is directly reflective of the community's needs rather than the perspectives of outsiders on the issue. Also, our survey has no unnecessary questions in order to respect the time and perspectives of the community members that will be filling out the form. Further, we thought of design justice in terms of the accessibility of our survey as we aim to have the survey in Japanese as well as English. This makes it more accessible to people who may not have English as their first language. Additionally, for our website, design justice can be implemented through some of the visual elements that will be incorporated. Being able to change the language of the website into another language, like Japanese, would allow for more inclusivity in terms of who can access and understand the map. Additionally, a design that makes the site accessible to people with color-blindness (e.g. color scheme of the map and its markers) and people with dyslexia (e.g. having the ability to change the font style) will also be considered to ensure that the site does not exclude these groups.

**Reply by XiongHer** [reply]
Your project seems pretty interesting, and I'm glad to hear that your team will go to Little Tokyo to collect the data from the community. It seems like you put a lot of thoughts and efforts into this data collection process to represent the community's voices. I also like your justice design and different aspects that you will include to ensure people can easily navigate and understand the information.

**Reply by willa-21** [reply]
It sounds like your project will take data justice and design justice into consideration, I'm excited to see the final project! I hope you are able to offer it in both languages, that seems like an important aspect for your community!

**Comment by kericsp** [comment]
While design justice is about ensuring equity of access to the tools we use and the systems we create using them (in software, online spaces, and in designing the physical spaces around us), data justice focuses more on equity in representation in the datasets that we use to determine whether our designs are representative and applicable for use by the wider world. I believe that without data justice, we cannot have true design justice, because it doesn't matter if we design the most perfect system for whatever it is we are trying to do, if we are lacking critical information or our information is inaccurate or misleading. In particular, the segment of the reading that highlighted how "thinking about bias and discrimination as... one-dimensional... ignores the structural conditions that underpin the problems that have been brought to light in the context of data systems", and how data and algorithms can both produce and reinforce some discursive frames over others (Dencik et al. 6). The danger here connects to our class discussions on solutions-oriented thinking and technological determinism; if we pursue easy fixes to these problems, we will be left with "little more than a set of reactionary technical solutions that ultimately fail to displace the underlying logic that produce unjust hierarchies of better and worse off subjects in the first place" (Dencik et al. 6). 

In relation to our group project, I think that we are still working through the stages of ensuring our final product will be just with respect to both data and design. In terms of data, we are trying to obtain our information from community members' voluntary participation, with respect to their privacy and discretion. In terms of design, we want to ensure accessibility to our target population by distributing our survey in Japanese as well as English, and ensure that our final project is navigable in Japanese. Ideally we will be able to provide accessibility features such as text-to-speech, large fonts on an easy-to-read background, and an intuitive design, in order to ensure that Japanese senior citizens will be able to navigate the site with or without a caregiver or aide.

**Reply by XiongHer** [reply]
I like your design justice approach and the way you describe how your team will make the website easy for Japanese elders to navigate as well as understand the information. The text-to-speech seems very cool, and it will be super helpful for the elders.

**Reply by penelopelamb** [reply]
I think that choosing to distribute your survey in both Japanese and English is a great priority and exemplifies prioritizing the target community. I also like your idea about text-to-speech and the large fonts. It's clear your group is making many considerations about how senior citizens may interact with a website differently than the general population.

**Comment by XiongHer** [comment]
Data justice and design justice are related as they both focus on centering the voices of the communities. Data justice seems to focus more on the collecting and analyzing process that try to include community’s voices and participations, while design justice seems to convey how data is used to represent the community. Both aim to ensure that researchers and firms that collected communities’ information used the data for good causes and to benefit the communities rather than oneself. These two methods will ensure technologies are used in ethical ways and be more transparent. 

For our group project, one way for me to use data justice is to make sure that our survey questions are focused on the community we want to represent and what kind of story they want to tell the world. It will share their stories rather than us telling their stories. As for design justice, we will have to make our page more interactive and easy to navigate so that the people can easily understand the experiences of graduate students as well as be able to find a community that they feel belong to and receive the necessary support that affirm their experiences as well.

**Reply by ashley-lu** [reply]
Your emphasis on focusing on the graduate student community through the survey questions and making an accessible design such that graduate students can easily share their experiences really represent what data justice and design justice are all about. I'm really excited to see your project!

**Comment by ntran27** [comment]
The connection between design and data justice lies in how they both aspire to center experiences of the communities at the core of any project. If data justice can informs our motivation and planning for a project, I think of design justice as the vehicle for how the project is carried out and delivered to the audience. Specific to our project, our group has considered these concepts to inform the types of questions we ask, how we design the website/survey, and ultimately how each story or user will be represented in the end. As we head into the final phases of our project, we are working to streamline everything so that users won't be bogged down by technical aspects of our website and can focus on the stories that are being shared by their peers. Also, we've worked to condense the survey and take out the solutions oriented questions.

**Reply by penelopelamb** [reply]
I agree with you about the similarities among design and data justice. Both of them choose to focus on the communities and their experiences. I think it is wise of your group to be considering design and data justice in every step of the process. Ensuring a functioning website and survey seems obvious but I like that you bring it up because I can see it being overlooked. My group is similarly working to shorten our survey for the same reason and to prevent survey fatigue.

**Reply by hajovel** [reply]
I agree with what you have said, and also really liked the analogy you made to design justice being the vehicle! I think that encapsulates it well, and shows how design justice can shape the view of one's opinion on a project based on its delivery. I like your group's approach to ensuring users are not overwhelmed by the technicalities of your website and more emphasized on the stories! That is something I will consider moving forward as well.

**Reply by angelineeow** [reply]
I really love the way you explain data justice and design justice! I also really like that your group project would focus on sharing the stories shared by the peers because if I were in the community's shoes, I would really appreciate learning about other experiences that are similar to mine. Great work!

**Comment by willa-21** [comment]
Data justice positions “data in a way that engages more explicitly with questions of power, politics, inclusion and interests, as well as established notions of ethics, autonomy, trust, accountability, governance and citizenship”(p. 874). Similar to design justice, it takes into account the intersection of ethics, equity, and systemic oppression. Both topics take into consideration the impact and importance of social justice and how data and design can be used for social good, beyond the surface level. Both are rooted in theory and practice, in order to challenge complacency when it comes to how we collect and utilize data, as well as design and the intersection of technology. As far as our group project, we are intentional in elevating the voices of our community and through our survey hope to collect data in a respectful and ethical way (we have kept it simple and straightforward in order to respect the time and privacy of our respondents). We do not believe the collection of data will cause any harm to our community. As far as our website design, we are continuing to reflect on how the website will be received by our community. We will prioritize simplicity and accessibility, in order to elevate the narratives of our participants.

**Reply by hajovel** [reply]
I enjoyed reading your post and particularly liked the point you made that data and design justice take account the intersectionality of ethics, equity, and systemic oppression and their goal in challenging this. I think that your project is focused on the community, such as how the website design will be received by your focused community, something important that I will also take to consideration for my group project!

**Comment by hajovel** [comment]
Data justice and design justice relate to each other as they both share a vital, common interest which is the emphasis of accessibility, ethical usage, and discovering ways in which data and design must be changed. In both of these, the focal point has been how data and design interact and affect other people. As data becomes increasingly essential to modern society due to the advancement of technology, so has its commodification and somewhat dehumanizing nature of the real users. Therefore, data justice calls for a similar approach to design justice where it refocuses its purpose on the people, and “decentralizes” technology. In doing so, data and design justice become important assets in realizing the contributions technology and design has on reimplementing negative impact on communities, such as “misrecognition and misrepresentation.”

In consideration of my group project, we are driven to include data and design justice in our project by several factors. In the design of our website, we want to make it accessible and highlight the contributors to our survey so the map is displayed and demonstrated as the focal point, with an inclusive color-blind friendly color scheme. Similarly, the survey will be a button that can be accessed while viewing the map. Our survey will highlight the stories of UCLA students and their experience with PrEP, considerate of surveyee fatigue and their time, and non-prescriptive.

**Reply by angelineeow** [reply]
I like that you mentioned that the data justice and design justice have contributed a negative impact on communities which made them so important. People need to always keep these concepts in mind when they attempt to empower those communities. I also really like that you guys are considering including a color-blind-friendly scheme on the website! Great job!

**Reply by bittertang** [reply]
I really liked how you incorporated points such as the commodification of technology from lecture into this post! I also really liked how you mention the different ways your group plans to uphold design justice not just through aesthetic design but also survey design, and it gave me ideas for how I want to incorporate design justice into my own project!

**Comment by angelineeow** [comment]
The concepts of data justice and design justice are related because they play essential roles in helping us understand social justice. Data justice is an approach focused on collecting the data for the community, while design justice is focused on delivering the voices of the community to the public. For our group,  we considered both concepts in drafting our group project by focusing on the questions to ask the survey respondents to optimize the information that we collected and make it worthwhile. We try to ask questions to listen to the community's opinion. We also focus on designing our website to make it accessible for the community that we would like to empower as this project focuses on assisting them. We would include some advice from the survey respondents and display it on our website in a way that we could provide a feeling of belonging to the community.

**Reply by ashley-lu** [reply]
The way you described the relation between data justice and design justice is something I didn't think about! I love the way you phrased it; both concepts are community-centered despite focusing on different actions.

**Reply by joonyloony** [reply]
i think that including a section for respondents to provide feedback is a great idea, since surveying is an active gathering of data you can actively change the survey to accommodate for certain issues that are raised as well as improve your accessibility of the website in general

**Reply by antonyblin** [reply]
I think it's great that you're taking so much time to optimize your questions for your community/audience. Framing your questions with as little bias or influence as possible is extremely important in founding data and design justice, and I like that you're considering this already.

**Comment by ashley-lu** [comment]
Data justice and design justice are similar in that they are deeply integrated with society, and people are constantly surrounded by data technologies (arising from the "datafication" of society) and design choices. As such, blindly making data or design choices can prove to be detrimental to " long-standing social, political, economic and cultural issues" with regards to reinforcing power imbalances and misrepresenting marginalized communities. Especially because jobs in data and design are popular these days, we need to be mindful of what social issues our actions exacerbate when we build a product that seems to help people on the surface level.

In our group project, we are in the process of making sure our mapplication aligns with data justice and design justice. We've gotten rid of many questions on our survey because we felt that they were not essential to our project and that they would needlessly increase survey fatigue. We also made sure not to gather any data that might be too sensitive or private. For the design of our website, we're making sure to put the community in the spotlight; the map will take up the majority of the mapplication, with a side section highlighting community voices. Our survey is linked to a button rather than embedded into the website so as to keep the focus on the community.

**Reply by joonyloony** [reply]
I totally agree with the survey fatigue point you raise, I've taken a ton of surveys in the past and sometimes it just feels like it won't end and it honestly decreases the quality of my responses as i'm trying to finish it up.

**Reply by NishuKrish** [reply]
I like how you are making sure to gather data that is not sensitive or private. This is especially important as people and communities are increasingly becoming distrustful of such practices due to large corporations misusing data for private gain. This, I think, may allay some of the concerns community members may have about taking your survey.

**Reply by kaittli** [reply]
The part about survey fatigue is a great reminder on how to build an effective survey—regardless of how good the questions are, answering a ton of questions is not favored by everyone. For the people that still complete surveys in these instances, less time may be spent on the questions towards the end because people may want to hurry up and finish.

**Comment by joonyloony** [comment]
I think data justice and design justice relate to each other as they both attempt to answer the ethical question of how to better serve marginalized communities through modern technological designs that have originally been used to (perhaps inadvertently) harm these communities and generate profits. Both highlight “how technology is situated in relation to larger systems of institutionalised oppression” and pose theory that give potential solutions, with design justice encouraging people to rethink the makeup/design of different structures, like UI/UX, and data justice encouraging people to think about how our data driven society may ignore structural issues that may skew the data/lead it to be misinterpreted which subsequently harms communities. I have definitely considered design justice when creating my project, even during practices where I am just making my own silly surveys, I think about inclusivity and the implications of each question’s response. Especially with relation to personal finances and its sensitive nature, I think that it is important to consider how the design of the survey as well as the data we collect and choose to share publicly, affect those who are surveyed as well as the community in general. We will be using design and data justice in order to create an accessible website and survey which protects the identities of those who are surveyed as well as being objective and not posing loaded questions in our surveys to potentially make those taking the survey uncomfortable or “nudge” our results in a specific direction.

**Reply by antonyblin** [reply]
I like that you brought up two very specific examples of UI/UX and our data-driven society for data and design justice. I'm especially glad that you are analyzing these grand ideas on even a smaller scale for your project, no matter how "silly" or "insignificant" it may seem. I think it's a great idea to focus on accessibility when designing your map/site!

**Reply by NishuKrish** [reply]
I like how you are incorporating data and design justice into your project by considering the community. It is really good that you are putting the community first and tailoring your site to their needs rather than your particular preferences.

**Reply by bittertang** [reply]
I liked how you describe the relationship between data justice and design justice, and how technology has been used to take advantage of marginalized communities as opposed to uplifting them. Before this class, I never thought of surveys as something that would be related in any way to design, so taking into consideration survey design and the implications of each question has definitely been an eye opening learning experience.

**Comment by kaittli** [comment]
Data justice and design justice are very much related to one another as both aim to support marginalized communities through making conditions more equitable and gives power back to these communities. There can be no design justice without data justice as data justice involves the collecting of information, stories, and voices that are to be represented and shared through design. Both complement one another in order to effectively tell authentic user stories. For the group project, we have incorporated numerous questions to ensure that many people of different perspectives can share their stories and through the use of conditional questions, are allowing for diverse perspectives. We want to make the design of our project as accessible as possible with intuitive interface as well as clear sections. The goal is to make a clean website that still captures the complexity of situation—utilizing the principles of data and design justice.
